qiskit/transpiler/passes/scheduling/alignments/reschedule.py

```diff
@@ -113,7 +114,7 @@ def _push_node_back(self, dag: DAGCircuit, node: DAGOpNode):
 
         if isinstance(node.op, Gate):
             alignment = self.pulse_align
-        elif isinstance(node.op, Measure):
+        elif isinstance(node.op, Measure) or isinstance(node.op, Reset):
```

Comment:
You can simplify this as `isinstance` lets you directly check against multiple types in a tuple:

```suggestion
        elif isinstance(node.op, (Measure, Reset)):
```